src/test/java/com/slack/kaldb/server/KaldbIndexerTest.java

```diff
@@ -144,12 +110,20 @@ public void testIndexFromKafkaSearchViaGrpcSearchApi() throws Exception {
     final LocalDateTime startTime = LocalDateTime.of(2020, 10, 1, 10, 10, 0);
 
     // Initialize kaldb config.
-    KaldbConfigs.KaldbConfig kaldbCfg = makeKaldbConfig("localhost:" + broker.getKafkaPort().get());
+    KaldbConfigs.KaldbConfig kaldbCfg =
+        KaldbConfigUtil.makeKaldbConfig(
+            "localhost:" + broker.getKafkaPort().get(),
+            8080,
+            TEST_KAFKA_TOPIC,
+            TEST_KAFKA_PARTITION,
+            KALDB_TEST_CLIENT,
+            TEST_S3_BUCKET,
+            8081);
     KaldbConfig.initFromConfigObject(kaldbCfg);
 
     // Create an indexer, an armeria server and register the grpc service.
     ServerBuilder sb = Server.builder();
-    sb.http(0);
+    sb.http(8080);
```

Comment:
I think hard coding the port number causes the unit tests to be flaky when tests are run in parallel.  So, I think leaving the port to be 0 and letting the library auto-assign a port may be more reliable.